src/main/java/com/arpnetworking/metrics/impl/TsdMetricsFactory.java

```diff
@@ -132,11 +132,25 @@ public static MetricsFactory newInstance(
      */
     @Override
     public Metrics create() {
-        return new TsdMetrics(
-                _serviceName,
-                _clusterName,
-                _hostName,
-                _sinks);
+        try {
```

Comment:
This seems to be an artifact of you filing a pull request from a branch that is ahead of your target (which is not master). Please explain what the purpose of this PR is or refile against master.